### ISSUE TITLE
Issue with module version 0.11

### ISSUE THREAD

**Issue by GraphR00t** [tommie/v8go#62]
The latest downloadable version is v0.10.1

```
$ go get github.com/tommie/v8go@v0.11
go: github.com/tommie/v8go@v0.11: no matching versions for query "v0.11"

$ go get github.com/tommie/v8go@latest
go: added github.com/tommie/v8go v0.10.1
....
```

On https://pkg.go.dev/github.com/tommie/v8go the latest version is `v0.10.1`

This is probably because the tag should be `v0.11.0` instead (https://semver.org).

> Thank you for maintaining this module by the way :)

**Comment by tommie** [comment]
Ah, you're right. Thanks for the bug report!

I've released v0.12.0. Waiting for pkg.go.dev to pick it up.

**Comment by tommie** [comment]
Seemed to work: https://pkg.go.dev/github.com/tommie/v8go@v0.12.0
